docs/content/TooltipV2.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-componentId: tooltip_v2
+componentId: tooltip
```

Comment:
Found out from Cameron that to be able to consolidate the versions of the same component on the new docs site, the ids need to be the same.